modules/base/rendering/screenspaceinsetblackout.cpp

```diff
@@ -0,0 +1,768 @@
+        "CopyToClipboard",
+        "Copy to clipboard",
+        "Copies the current configuration to the clipboard so that it can be pasted "
+        "into the asset file.",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CalibrationPatternInfo = {
+        "EnableCalibrationPattern",
+        "Enable Calibration Pattern",
+        "Enables the calibration pattern.",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CalibrationColorInfo = {
+        "EnableCalibrationColor",
+        "Enable Calibration Color",
+        "Set Blackout Shape to a bright color for easier calibration.",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo AddControlPointInfo = {
+        "AddControlPoint",
+        "Add Control Point",
+        "Adds a new control point to the spline.",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo RemoveControlPointInfo = {
+        "RemoveControlPoint",
+        "Remove Control Point",
+        "Removes the selected control point from the spline.",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo RemoveSelectorInfo = {
+        "RemoveSelector",
+        "Select Point To Remove",
+        "Removes the selected control point.",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo AddSelectorInfo = {
+        "AddSelector",
+        "Select Where To Add",
+        "Select where to add a new point.",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo NewPointPositionInfo = {
+        "NewPointPosition",
+        "Point Position",
+        "X and Y coordinates for where to add the new control point.",
+        openspace::properties::Property::Visibility::User
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CalibrationTextureInfo = {
+        "CalibrationTexture",
+        "Calibration Texture",
+        "Texture used as calibration pattern.",
+        openspace::properties::Property::Visibility::Developer
+    };
+
+    // A ScreenSpaceInsetBlackout can be used to render a screen-space shape used to 
+    // black out part of the renderin. This can be useful in a dome environemnt where 
+    // you have a secondary presentation projector that can project on the dome surface. 
+    // This renderable can be used to black out some part of the dome OpenSpace rendering 
+    // in order to avoid overlapping rendering between the dome projectors and the
+    // presentation projector.
```

Comment:
This is a bit repetitive. How about:
```suggestion
    // The blackout is used to avoid overlapping rendering between the dome projectors and the
    // presentation projector.
```
?